### PR TITLE
Bad wording and example fixes in cache documentation.

### PR DESCRIPTION
**Pull request by fredericDelaporte**

### CODE REVIEW

File: doc/reference/modules/nhibernate_caches.xml [hunk 1]
```diff
@@ -209,7 +209,7 @@
 	<section id="NHibernate.Caches.SysCache">
 		<title>SysCache Configuration</title>
 		<para>
-			As SysCache relies on <classname>System.Web.Caching.Cache</classname> for the underlying implementation.
+			SysCache relies on <classname>System.Web.Caching.Cache</classname> for the underlying implementation.
 			Following NHibernate configuration settings are available:
```
Comment (gliljas) [inline doc/reference/modules/nhibernate_caches.xml]: The following

File: doc/reference/modules/nhibernate_caches.xml [hunk 2]
```diff
@@ -568,9 +568,9 @@
 	</section>
 
 	<section id="NHibernate.Caches.RtMemoryCache">
-		<title>Runtime Memory Configuration</title>
+		<title>RtMemoryCache Configuration</title>
 		<para>
-			As runtime memory relies on <classname>System.Runtime.Caching.MemoryCache</classname> for the
+			RtMemoryCache relies on <classname>System.Runtime.Caching.MemoryCache</classname> for the
 			underlying implementation.
 			Following NHibernate configuration settings are available:
```
Comment (gliljas) [inline doc/reference/modules/nhibernate_caches.xml]: The following

Comment (fredericDelaporte) [reply doc/reference/modules/nhibernate_caches.xml]: I frequently read articles where I find similar article omissions. Maybe I am missing the rule allowing their omission, or maybe that is an american thing (I am reading mainly an american paper, IEEE Spectrum). I would be glad if you can tell me more about that.

**Comment by fredericDelaporte** [comment]
Changed.

**Comment by fredericDelaporte** [comment]
Thanks Gunnar,
For a PR with a single commit, doing a "Rebase and merge" is less noisy than a "Create a merge commit". It avoids the merge commit which has not much interests when the PR was only having one commit.

**Comment by gliljas** [comment]
Ah, sorry, I didn't know that was an option. Rebasing is definitely a better option. Will keep it in mind, thanks!

**Comment by fredericDelaporte** [comment]
I am just following Alexander's recommendations he told me on a previous PR: https://github.com/nhibernate/nhibernate-core/pull/582#issuecomment-290379985

**Comment by gliljas** [comment]
Sounds reasonable

**Pull request merged**